src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuilder.java

```diff
@@ -133,7 +133,7 @@ private int getScmChange(AbstractProject subjob, PhaseJobsConfig phaseConfig, Ab
 
     public boolean evalCondition(final String condition, final AbstractBuild<?, ?> build, final BuildListener listener) {
         try {
-            return (Boolean) Eval.me(expandToken(condition, build, listener).toLowerCase().trim());
+            return (Boolean) Eval.me(expandToken(condition, build, listener).trim());

```

Comment:
I'm not certain that the conditional functionality is usable while being lower cased.  Most groovy code won't evaluate without correct capitalization.